GUI/DeusEx/Classes/HUDMedBotNavBarWindow.uc

```diff
@@ -0,0 +1,59 @@
+class DXRHUDMedBotNavBarWindow injects #var(prefix)HUDMedBotNavBarWindow;
+
+function CreateButtons() {}
+
+function CreateAllButtons()
+{
+    Super.CreateButtons();
+}
+
+function CreateExitButton()
+{
+    Super(#var(prefix)PersonaNavBarBaseWindow).CreateButtons(); // don't create "Health" or "Augmentations" buttons
+}
+
+function bool ButtonActivated(Window buttonPressed)
```

Comment:
why is this function needed? I see it's slightly different but not sure of the reason, could use some comments

this one only invokes the new ui screen when there's an existing window, while the vanilla code doesn't care, I guess you're fixing an issue if you keep clicking the button multiple times?

Reply:
The key difference is that the vanilla code, when the "Health" or "Augmentations" buttons are clicked:
1) creates a new screen object, even if it's the one same one you're already on
2) only calls `SetMedicalBot` on it if it's not the screen class you're already on, thus potentially skipping the new button/label logic
The end result is that the navbar buttons disappear if you click the button for the screen you're already on. Making `ButtonActivated` always call `SetMedicalBot`, regardless of the current screen, does solve the problem. But a simpler solution becomes apparent, which is to just ignore button clicks for the screen the player is already on.